inst/pages/phyloseq_cheatsheet.qmd

```diff
@@ -0,0 +1,476 @@
+# phyloseq vs TreeSE cheatsheet {#sec-extras}
+
+```{r}
+#| label: setup
+#| echo: false
+
+knitr::opts_chunk$set(eval=TRUE, warning=FALSE, message=FALSE)
+```
+
+This section has a cheatsheet for translating common functions in phyloseq
+to TreeSE/mia with example code.
+
+```{r}
+#| label: download_libs
+
+# Download libraries
+library(mia)
+library(phyloseq)
+library(dplyr)
+library(ggplot2)
+```
+
+Start by loading data as a `phyloseq` object "phy" and as `TreeSE` object "tse".
+
+```{r}
+#| label: load_data
+
+# Loading example data
+# Using GlobalPatterns dataset
+
+data(package = "phyloseq", "GlobalPatterns") # phyloseq object
+phy <- GlobalPatterns # Rename
+phy # Check the phyloseq object
+
+data(package = "mia", "GlobalPatterns") # TreeSE object
+tse <- GlobalPatterns # Rename
+tse # Check the tse object
+```
+
+### Accessing different types of data in `phyloseq` versus `TreeSE`
```

Comment:
(Final) thing: This is now subsubheader  (3 "#"s), you could make it subheader (2 "#"s)